adafruit_wiznet5k/adafruit_wiznet5k_socket.py

```diff
@@ -255,30 +360,40 @@ def connect(self, address, conntype=None):
             raise RuntimeError("Failed to connect to host", host)
         self._buffer = b""
 
-    def send(self, data):
-        """Send data to the socket. The socket must be connected to
-        a remote socket.
+    def send(self, data: Union[bytes, bytearray]) -> None:
+        """
+        Send data to the socket.
 
-        :param bytearray data: Desired data to send to the socket.
+        The socket must be connected to a remote socket.
+
+        :param bytearray data: Data to send to the socket.
         """
         _the_interface.socket_write(self.socknum, data, self._timeout)
         gc.collect()
 
-    def sendto(self, data, address):
-        """Send data to the socket. The socket must be connected to
-        a remote socket.
+    def sendto(self, data: bytearray, address: [Tuple[str, int]]) -> None:
+        """
+        Connect to a remote socket and send data.
 
-        :param bytearray data: Desired data to send to the socket.
+        :param bytearray data: Data to send to the socket.
         :param tuple address: Remote socket as a (host, port) tuple.
         """
         self.connect(address)
         return self.send(data)
 
-    def recv(self, bufsize=0, flags=0):  # pylint: disable=too-many-branches
-        """Reads some bytes from the connected remote address.
+    def recv(
+        # pylint: disable=too-many-branches
+        self,
+        bufsize: int = 0,
+        flags: Any = 0,
```

Comment:
I think it'd be best to make the type declaration here of `flags` match the one in the doc strings. I don't know that it matters much if we use `int` or `Any` since it's ignored. Can you change one or the other so they are consistent.